### PR TITLE
Fix/backend info

### PR DESCRIPTION
**Pull request by avilchess**
Make sure you have checked _all_ steps below.

### Description
- [ ] Here are some details about my PR, including screenshots of any UI changes:

This PR fix the function backend_info()

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits have been squashed if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

## License
- [ ] Add a [Mozilla Public License 2.0 license header](http://mozilla.org/MPL/2.0/) to all the new files.

### Documentation
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.

### CODE REVIEW

File: tests/unit_tests/library_unit_tests.py [hunk 1]
```diff
@@ -64,8 +64,21 @@ def test_get_device(self):
                 self.assertEqual(get_device_id(), i)
 
     def test_version(self):
-        v = version()
-        self.assertEqual(v, '0.1.0')
+        self.assertEqual(version(), self.get_khiva_version_from_github())
```
[the remark this answers is not in the document]
Comment (avilchess) [reply tests/unit_tests/library_unit_tests.py]: You are right, but it is still better than having the version as a constant string.

File: .CI/travis/install-khiva.sh [hunk 2]
```diff
@@ -5,20 +5,67 @@
...
 
-    sudo ldconfig
+
+    # Cloning Github repo into khiva-library folder
+    git clone https://github.com/shapelets/khiva.git khiva-library
+    cd khiva-library
+    mkdir -p build && cd build
+    if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        conan install .. -s compiler=apple-clang -s compiler.version=9.1 -s compiler.libcxx=libc++ --build missing
+        cmake .. -DKHIVA_ENABLE_COVERAGE=ON -DKHIVA_ONLY_CPU_BACKEND=ON -DKHIVA_BUILD_DOCUMENTATION=OFF -DKHIVA_BUILD_EXAMPLES=OFF -DKHIVA_BUILD_BENCHMARKS=OFF
```
Comment (otorreno) [inline .CI/travis/install-khiva.sh]: I would set KHIVA_ENABLE_COVERAGE to OFF

File: .appveyor.yml [hunk 3]
```diff
@@ -7,13 +7,14 @@
...
 environment:
+    GENERATOR: "Visual Studio 15 Win64"
     BOOST_ROOT: C:\Libraries\boost_1_67_0
     AF_PATH: C:\Program Files\ArrayFire\v3
     KHIVA_PATH: C:\Program Files\Khiva\v0
     matrix:
         - PATH: $(AF_PATH)\lib;$(KHIVA_PATH)\lib;C:\Python36-x64;C:\Python36-x64\Scripts;$(PATH)
-        - PATH: $(AF_PATH)\lib;$(KHIVA_PATH)\lib;C:\Python35-x64;C:\Python35-x64\Scripts;$(PATH)
 cache:
+    - c:\tools\vcpkg\installed\
```
Comment (otorreno) [inline .appveyor.yml]: remove trailing slash

File: .appveyor.yml [hunk 4]
```diff
@@ -23,8 +24,15 @@ before_build:
     - ps: if(!(Test-Path -Path "C:\Program Files\ArrayFire" )){ appveyor DownloadFile "https://www.dropbox.com/s/ndt1eaouriiu4qi/ArrayFire-v3.5.1.zip?dl=1" -FileName ArrayFire-v3.5.1.zip }
     - ps: if(!(Test-Path -Path "C:\Program Files\ArrayFire" )){ 7z x ArrayFire-v3.5.1.zip -o"C:\Program Files" }
     - reg add HKCU\Software\Kitware\CMake\Packages\ArrayFire /v ArrayFire_CMake_DIR /d "C:\Program Files\ArrayFire\v3\cmake" /f
-    - ps: if(!(Test-Path -Path "C:\Program Files\Khiva" )){ appveyor DownloadFile "https://github.com/shapelets/khiva/releases/download/v0.1.0/khiva-v0.1.0-unattended.exe" -FileName khiva-v0.1.0-unattended.exe }
-    - ps: if(!(Test-Path -Path "C:\Program Files\Khiva" )){ .\khiva-v0.1.0-unattended.exe /S }
+    - vcpkg install --triplet x64-windows gtest eigen3
+#   - ps: if(!(Test-Path -Path "C:\Program Files\Khiva" )){ appveyor DownloadFile "https://github.com/shapelets/khiva/releases/download/v0.1.0/khiva-v0.1.0-unattended.exe" -FileName khiva-v0.1.0-unattended.exe }
```
Comment (otorreno) [inline .appveyor.yml]: why not the same as in travis, the KHIVA_INSTALLER variable and do one or the other installation option?

**Pull request merged**